.gitmodules

```diff
@@ -1,12 +1,12 @@
 [submodule ".github/actions/get-workflow-origin"]
-	path = .github/actions/get-workflow-origin
-	url = https://github.com/potiuk/get-workflow-origin
+    path = .github/actions/get-workflow-origin
```

Comment:
Tabs are not bad characters -- flake8/black deals with this, so leave tabs alone.

Reply:
Or ignore the following files like we did before:

```
        exclude: ^airflow/_vendor/|^clients/gen/go\.sh$|^\.gitmodules$
```

Reply:
we alredy had "forbid tabs" before - I just missed an exclusion (which I have to add anyway).

Reply:
(this one will be better because it will fix them automatically).